### ISSUE TITLE
Supporting 'Custom Roles for TTT' addon

### ISSUE THREAD

**Issue by AgeOfPlain** [Tommy228/tttdamagelogs#382]
I know this is an enhancement but I don't know where else to put it.

Would it be possible to get a version or con-var that lets RDM Manager support this : https://steamcommunity.com/sharedfiles/filedetails/?id=1215502383

As we can't find any Logs that support it and include a reporting system, we even tried Blogs and that started lagging the server loads so we had to remove it. 

Anyway yea it would be great if there could be a version that supports that because at the current time it just doesn't work.

**Comment by Alf21** [comment]
There won‘t be any support for this addon since this author is inactive. I created a compatible version for the TTT2 add-on on my own. This also didn‘t got merged. So there is just one way to do so: Make a conptaible version on your own with a fork of this repo or wait for an undefined time.

**Comment by Butanal** [comment]
Yes, currently the only edits to the repo are bugfixes. If you are willing to write it or find someone who can write it, I might be able to take a look and merge, but cannot provide support for this myself.
